apps/registry.json

```diff
@@ -232,5 +232,28 @@
         ],
         "path": "",
         "defaultPassword": ""
+    },
+    {
+        "id": "lndhub",
+        "category": "Wallet Servers",
+        "name": "BlueWallet Server",
+        "version": "1.3.0",
+        "tagline": "Connect a BlueWallet to your Umbrel",
+        "description": "Wrapper for Lightning Network Daemon. It provides separate accounts and trust minimization for end users.",
```

Comment:
"bla bla onboard your family and friends to Lightning powered by your own node"

cc @ncoelho